ui/components/privacy-view.jsx

```diff
@@ -0,0 +1,145 @@
+import React from 'react';
+
+export default function PrivacyView() {
+  return (
+    <div className="ml3">
+      <h2 className="h1">Privacy Policy</h2>
```

Comment:
I was following the copy and pasted source. Will change to match the 18F way!
Changed!